components/org.wso2.identity.event.websubhub.publisher/src/main/java/org/wso2/identity/event/websubhub/publisher/service/WebSubEventSubscriberImpl.java

```diff
@@ -70,90 +68,92 @@ public String getName() {
     }
 
     @Override
-    public void subscribe(List<String> channels, String eventProfileVersion, String endpoint, String secret,
-                          String tenantDomain) throws WebhookMgtException {
+    public List<Subscription> subscribe(List<String> channels, String eventProfileVersion, String endpoint,
```

Comment:
Can't we just pass the webhook itself here

Reply:
I actually think it's better to pass the webhook model.
This makes the contract clean
And also in future is there are more subscription related properties introduced we don't need to modify the contract

Reply:
fixed with https://github.com/wso2-extensions/identity-event-publishers/pull/25/commits/67d32091145f93afc04a6815cd9e2ef1828a00c3

but for the eventProfileVersion is not in the webhook. It's in the IDN_WEBHOOK_TOPICS table. May I add it to the webhook table also?